administrator/components/com_cpanel/tmpl/cpanel/default.php

```diff
@@ -33,6 +33,11 @@
 	$cols = 0;
 	foreach ($this->modules as $module)
 	{
+	    // hack to fix loading page with new factory in com_content
+	    if ($module->id == '3' || $module->id == '4')
```

Comment:
?

Reply:
This is just a hack to load the page. As it tries to instantiate the ArticleModel with the old factory.